views/apartments/edit.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Update Information for <%= @apartment.address %></h1>
```

Comment:
if you want to keep working on this - try using partials to share code between new and edit views - http://www.sinatrarb.com/faq.html#partials